AutoOption.cmake

```diff
@@ -0,0 +1,45 @@
+# Add a new option with default value "auto":
+#  autoOption(FOOBAR "Support foobar")
+#
+# To check if FOOBAR support should be enabled, use:
+#   if(FOOBAR)
+#
+# If necessary dependencies for FOOBAR are missing, use:
+#   autoDisable(FOOBAR "Did not find baz")
+#
+# Example:
+#  autoOption(FOOBAR "Support foobar")
+#   if(FOOBAR)
+#     Check for FOOBAR here.
+#     if(NOT BAZ_FOUND)
+#       autoDisable(FOOBAR "Did not find baz")
+#     endif()
+#   endif()
+#
+# If necessary dependencies for FOOBAR are missing, use:
```

Comment:
```suggestion
```

Reply:
Huh? These two lines should just be removed?

Reply:
Yes.. the same are above.

Reply:
Oh, thanks, I wonder why I didn't notice this duplication...